instant.ts

```diff
@@ -28,14 +28,13 @@ interface PackagesMap {
 
 type EnvironmentVar = {
   'Environment Variable': string
-  'Default Value': string
-  'Updated Value': string | undefined
+  'Current Value': string | undefined
 }
 
 function getInstantOHIEPackages(): PackagesMap {
```

Comment:
Can we add a check on the package-metadata.ts structure ?

Reply:
The Zod library offers a good way to validate js objects.

Reply:
We already had a json schema file for it so i just added ajv which can validate that schema.